src/libMediator/Mediator.h

```diff
@@ -99,6 +99,16 @@ class Mediator
     void UpdateTxBlockRand(bool isGenesis = false);
 
     std::string GetNodeMode(const Peer& peer);
+
+    /// Initialize heart beat checking
+    void HeartBeat_Init();
+
+    /// Trigger heart beat
+    void HeartBeat();
+
+private:
+    std::mutex m_heartBeatMutex;
+    uint64_t m_heartBeatTime;
```

Comment:
why is this a uint64_t int? I don't see this going beyond 3600

Reply:
ok, I will change to unsigned int